control/controlclient/direct.go

```diff
@@ -609,6 +613,10 @@ func (c *Direct) PollNetMap(ctx context.Context, maxPolls int, cb func(*NetworkM
 		if resp.KeepAlive {
 			continue
 		}
+
+		undeltaPeers(&resp, previousPeers)
+		previousPeers = resp.Peers
```

Comment:
I think it's easier to reason about if you put the `previousPeers = cloneNode(resp.Peers)` here instead of below.

Reply:
Done.